perma_web/perma/views/common.py

```diff
@@ -383,3 +383,29 @@ def contact_thanks(request):
     """
     registrar = Registrar.objects.filter(pk=request.GET.get('registrar', '-1')).first()
     return render(request, 'contact-thanks.html', {'registrar': registrar})
+
+
+def robots_txt(request):
+    """
+    robots.txt
+    """
+    from ..urls import urlpatterns
+
+    # redundant -- from models.py
+    guid_character_set = "23456789ABCDEFGHJKLMNPQRSTUVWXYZ"
```

Comment:
Awesome! My one suggestion is to put this in `Link.GUID_CHARACTER_SET` and refer to it from both places to avoid the redundancy.